integration-test/pages.spec.js

```diff
@@ -0,0 +1,110 @@
+import { join, resolve } from 'path'
+import { launch } from 'puppeteer'
+import { toMatchImageSnapshot } from 'jest-image-snapshot'
+import next from 'next'
+
+import docs from '../pages/docs.json'
+
+expect.extend({ toMatchImageSnapshot })
+jasmine.DEFAULT_TIMEOUT_INTERVAL = 200000
+
+const globalCss = `
+  * {
+    -webkit-animation: unset !important;
+    animation: unset !important;
+    font-family: 'Arial' !important;
+    font-smooth: never !important;
+  }
+
+  pre, pre * {
+    font-family: 'Courier' !important;
+  }
+
+  .hero-header {
+    min-height: auto !important;
+  }
+`
+
+const PORT = process.env.PORT || 12345
+
+describe('Pages', () => {
+  let errors = []
+  let browser
+  let page
+  let app
+
+  beforeAll(async () => {
+    app = next({
+      dir: '.',
+      dev: false,
+      quiet: true,
+      conf: {}
+    })
+
+    await app.start(PORT, 'localhost')
+    browser = await launch({
+      executablePath: process.env.GOOGLE_CHROME_BINARY || undefined,
+      args: ['--no-sandbox', '--disable-setuid-sandbox', '--disable-lcd-text']
+    })
+    page = await browser.newPage()
+
+    page.on('request', req => {
+      if (!req.url.startsWith(`http://localhost:${PORT}`)) {
+        console.log(req.url)
```

Comment:
console.log?

Reply:
ah darn it; I'll remove it once I squash and rebase 😆